ui/src/Signin.tsx

```diff
@@ -74,13 +75,28 @@ export class Signin extends PureComponent<Props, State> {
   private checkForLogin = async () => {
     try {
       await client.users.me()
+      const redirectIsSet = !!window.localStorage.getItem('redirectTo')
```

Comment:
do we have a way to access storage without hitting `localStorage` directly? maybe some kind of module that handles fallbacks in case localStorage isn't enabled?

Reply:
It seems like we're setting/getting state items here:

https://github.com/influxdata/influxdb/blob/master/ui/src/localStorage.ts#L41-L51

But they make use of the same underlying functionality. Is it beneficial to abstract some of the logic to a generalized setter in that file? I'm curious to know

Reply:
yeah, we should use the abstractions that exist in the system, even if they're doing the same thing. if we needed to add an extra step (like let's say we have to add a timestamp to all writes to local storage), we'd only have to update the code to do that in one place, if everything made use of that function. it handles error messaging in a consistent way as well

Reply:
Sounds good. I'll go ahead and write up some utility functions and send up a commit for that